Libraries/Image/ImageSourcePropType.js

```diff
@@ -43,6 +43,32 @@ const ImageURISourcePropType = PropTypes.shape({
    */
   body: PropTypes.string,
   /**
+   * `cache` determines how the requests handles potentially cached
+   * responses.
+   *
+   * - `default`: Use the native platforms default strategy. `useProtocolCachePolicy` on iOS.
+   *
+   * - `reload`: The data for the URL will be loaded from the originating source.
+   * No existing cache data should be used to satisfy a URL load request.
```

Comment:
I do not personally have a use case for it, however I implemented it for completeness since it exists both in iOS and in `fetch`. The only use case I can think of is when the server or intermediate proxies can't be trusted to set a appropriately low cache timeout for volatile content.